src/main/java/org/dcsa/uisupport/controller/TimestampExportController.java

```diff
@@ -37,8 +40,12 @@
     private final R2dbcDialect r2dbcDialect;
     private final static Set<Class<? extends Event>> OPERATIONS_EVENT_TYPE = Set.of(OperationsEvent.class);
     private final DataExportDefinition<OperationsEvent> dataExportDefinition = DataExportDefinition.<OperationsEvent>builder()
-            .column("Publisher SMDG code", (oe) -> null) // TODO <--
+            .column("Publisher SMDG code", oe -> mapPublisher(oe, p -> p.getIdentifyingCodes().stream()
+                    .filter(idc -> idc.getCodeListResponsibleAgencyCode().equals(CodeListResponsibleAgency.SMDG.getCode()))
+                    .map(PartyTO.IdentifyingCode::getPartyCode)
+                    .collect(Collectors.toList())))

```

Comment:
Please generate a string here.  This is output to Excel and a list is not well-defined as output.